Data/FE13/UI/Modules/Job.yml

```diff
@@ -82,6 +86,22 @@ top_level:
               - fid
     - type: tabs
       tabs:
+        - title: Misc. && Unknown
```

Comment:
I don't like having this as the first tab. Maybe we could put the bmap icon fields into the "Overview" box instead?